javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithMembers.java

```diff
@@ -53,10 +54,22 @@ public default FieldDeclaration addField(Class<?> typeClass, EnumSet<Modifier> m
      * @return the {@link FieldDeclaration} created
      */
     public default FieldDeclaration addField(String type, EnumSet<Modifier> modifiers, String name) {
+        return addField(new ClassOrInterfaceType(type), modifiers, name);

```

Comment:
Maybe use "Modifier... modifiers" here? Or maybe that's better for another issue. This PR is big enough as it is.